FiveCalls/FiveCalls/ScheduleRemindersController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ScheduleRemindersController.swift
+//  FiveCalls
+//
+//  Created by Christopher Brandow on 2/8/17.
+//  Copyright © 2017 5calls. All rights reserved.
+//
+
+import Foundation
+
+class ScheduleRemindersController: UIViewController {
+
+    @IBOutlet weak var timePicker: UIDatePicker!
+    @IBOutlet weak var daysOfWeekSelector: MultipleSelectionControl!
+
+    var notificationsChanged: Bool?
```

Comment:
Nit: This immediately get's set to false in viewDidLoad, so it could probably be
```swift
var notificationsChanged: Bool = false
```

this also simplifies the guard in viewWillDisappear to

```swift
guard notificationsChanged == true else { return }
```